stat/distuv/statdist.go

```diff
@@ -0,0 +1,64 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math"
+
+	"gonum.org/v1/gonum/mathext"
+)
+
+// KullbackLiebler is a type for computing the Kullback-Leibler divergence from l to r.
+//
+// The Kullback-Liebler divergence is defined as
+//  D_KL(l || r ) = \int_x p(x) log(p(x)/q(x)) dx
+// Note that the Kullback-Liebler divergence is not symmetric with respect to
+// the order of the input arguments.
+type KullbackLeibler struct{}
+
+// DistBeta returns the KullbackLeibler distance between Beta distributions
```

Comment:
Is it a "distance" or "divergence"? Mathematically one expects certain properties from a distance which this does not seem to have.